### ISSUE TITLE
incorrect farmer and pool reward address detected

### ISSUE THREAD

**Issue by UniquePeteA** [joaquimguimaraes/farmr#124]
farmr.exe for windows lists the wrong farmer and pool reward address when I choose option 2 to view my addresses.  Then it generates an error message continuously repeating that the pool reward address does not match any of my hot/cold wallet addresses.

**Comment by joaquimguimaraes** [comment]
It gets farmer and pool reward addresses directly from chia RPC service. Are you sure these are wrong?

**Comment by UniquePeteA** [comment]
Perhaps it's from an older set of keys or something.  I originally had
multiple key sets and deleted one. "chia keys show", "chia wallet show" and
"chia plotnft show" do not show anything resembling what farmr.exe shows
for Pool and Farmer rewards address.  Neither does the chia GUI.

**Comment by joaquimguimaraes** [comment]
It's not based on your keys at all. What platform are you using? macOS?

**Comment by joaquimguimaraes** [comment]
You should look for the pool and farmer rewards in chia's config.yaml
Its path is if I recall correctly
This is where your block rewards are sent to
.chia/mainnet/config/config.yaml

**Comment by UniquePeteA** [comment]
Yes, you are correct.  I see it as the xch_target_address under both pool
and farmer in that file.  I'll figure out where that came from.  Thanks.
